.github/workflows/run_tests.yml

```diff
@@ -72,6 +72,12 @@ jobs:
       uses: actions/setup-node@56899e050abffc08c2b3b61f3ec6a79a9dc3223d # v1.4.4
       with:
         node-version: "14.x"
+    - name: 'Rust toolchain'
```

Comment:
The `rust_binaries` test use this toolchain, which pulls the latest stable version of rust. Otherwise we rely on the assumption that the `ubuntu-latest` container includes the same rust version as the toolchain (which is not always the case)